packages/cms/.gitignore

```
@@ -5,8 +5,11 @@
 /.pnp
 .pnp.js
 
-# build
+# build and data exports
 dist
+export
+development.tar.gz
+production.tar.gz
```

```suggestion
*.tar.gz
```

I think that works as well

---

I think this has been resolved in the branch already.